src/route-handlers/describe-workflow/describe-workflow.ts

```diff
@@ -14,7 +19,7 @@ export default async function describeWorkflow(
   const decodedParams = decodeUrlParams(requestParams.params);
 
   try {
-    const res = await grpcClient.clusterMethods[
+    const describeResponse = await grpcClient.clusterMethods[
```

Comment:
nit: describeResponse is (ironically) not very descriptive
```suggestion
    const describeWorkflowResponse = await grpcClient.clusterMethods[
```